lib/public/views/DataPasses/format/formatDataPassName.js

```diff
@@ -22,24 +22,11 @@ import { h, iconDelete } from '/js/src/index.js';
  * @return {Component} data pass name display
  */
 export const formatDataPassName = ({ name, versions }) => {
-    if (versions?.every(({ deletedFromMonAlisa }) => deletedFromMonAlisa)) {
-        return badge(
-            h('.flex-row.g1', [
-                tooltip(
-                    iconDelete(),
-                    'The production was removed from MonALISA',
-                ),
-                name,
-            ]),
-            { color: 'var(--color-success)' },
-        );
-    } else if (versions?.some(({ deletedFromMonAlisa }) => deletedFromMonAlisa)) {
+    const allDeleted = versions?.every(({ statusHistory }) => statusHistory[statusHistory.length - 1].status === DataPassVersionStatus.DELETED);
+    if (allDeleted) {
         return h('.flex-row.g1', [
-            tooltip(
-                iconDelete(),
-                'Part of the production was removed from MonALISA',
-            ),
-            h('.success', name),
+            tooltip(h('.gray-darker', iconDelete()), 'The production was removed from MonALISA'),
```

Comment:
I think that here we should remove this icon, it's not necessary anymore. Same bellow.

Reply:
applied